docs/deployment-examples/custom-scripts/index.md

```diff
@@ -11,8 +11,8 @@ As a convention-oriented deployment tool, Octopus can perform a number of action
 Octopus supports the following scripting environments:
 
  - PowerShell scripts (.ps1)
- - C# scripts (.csx) using [ScriptCS](https://github.com/scriptcs/scriptcs)
```

Comment:
A little hat-tip to Linux being more first-class. I wonder if we should change it any more, aligning it with the telemetry we see? Most popular at the top.

Reply:
That makes sense to me; it would mean making things easier for the most number of users.